src/rgw/rgw_rados.cc

```diff
@@ -2847,10 +2847,12 @@ class RGWRadosThread {
 
   atomic_t down_flag;
 
+  string thread_name;
+
   virtual uint64_t interval_msec() = 0;
   virtual void stop_process() {}
 public:
-  RGWRadosThread(RGWRados *_store) : worker(NULL), cct(_store->ctx()), store(_store) {}
+  RGWRadosThread(RGWRados *_store) : worker(NULL), cct(_store->ctx()), store(_store), thread_name("radosgw") {}
```

Comment:
i have a similar patch in another pr, see https://github.com/ceph/ceph/pull/13111/commits/9462afa339a9da4a3aae91a2c7d791b7bfc591ab as an example